uniprotkb-rest/src/test/resources/application.properties

```diff
@@ -44,5 +44,9 @@ streamer.uniprot.batchSize=1000
 streamer.uniprot.valueId=accession_id
 streamer.uniprot.defaultsField=avro_bin
 
+solr.viewby.uniprotCollection=uniprot
+solr.viewby.ecDir=/my/ec/dir
+solr.viewby.uniPathWayFile=/my/unipathway/file
+
```

Comment:
missing solr.viewby.keywordFile=

Have you created tests for it? do you think it is worst create it?

Reply:
Great spot.

Yeah I was thinking about tests and I originally wrote some tests, but I ended up testing whether Spring's @ConfigurationProperties was working :) which seemed a bit wrong. So, I think we should just assume that library just works. Are you happy with this?